mdm/orientdb-api/src/main/java/com/gentics/mesh/core/data/MeshCoreVertex.java

```diff
@@ -20,6 +20,7 @@
 	 * @param batch
 	 *            Batch to which entries will be added in order to update the search index.
 	 * @return true if the element was updated. Otherwise false
+	 * @deprecated move to DAO or rename
```

Comment:
We should restrict direct entity changes - everything has to use corresponding DAO.